docs/mr_proc/workflow/dicom_org.md

```diff
@@ -2,40 +2,49 @@
 
 ---
 
-This is a dataset specific process and needs to be customized based on local scanner DICOM dumps and file naming. This organization should produce, for a given session, participant specific dicom dirs. Each of these participant-dir contains a flat list of dicoms for the participant for all available imaging modalities and scan protocols.
+This is a dataset specific process and needs to be customized based on local scanner DICOM dumps and file naming. This organization should produce, for a given session, participant specific dicom dirs. Each of these participant-dir contains a flat list of dicoms for the participant for all available imaging modalities and scan protocols. The manifest is used to determine which new subject-session pairs need to be processed, and a `doughnut.csv` file is used to track the status for the DICOM reorganization and BIDS conversion steps.
 
 ---
 ### Key directories and files
 
+- `<DATASET_ROOT>/tabular/demographics/mr_proc_manifest.csv`
 - `<DATASET_ROOT>/downloads`
 - `<DATASET_ROOT>/scratch/raw_dicoms`
+- `<DATASET_ROOT>/scratch/raw_dicoms/doughnut.csv`
 - `<DATASET_ROOT>/dicom`
-- `<DATASET_ROOT>/tabular/demographics/mr_proc_manifest.csv`
 
 ### Procedure
 
-- Download DICOM dumps (e.g. ZIPs / tarballs) in the `<DATASET_ROOT>/downloads` directory. It is recommended that different visits (i.e. sessions) are downloaded in separate sub-directories and named as listed in the `global_config.json`.
-- Extract (and rename* if needed) all participants into `<DATASET_ROOT>/scratch/raw_dicoms` separately for each visit (i.e. session). 
-  
+1. Run [`workflow/dicom_org/check_dicom_status.py`](https://github.com/neurodatascience/mr_proc/blob/main/workflow/dicom_org/check_dicom_status.py) to update `doughnut.csv` based on the manifest. It will add new rows for any subject-session pair not already in the file.
+    - To create the `doughnut.csv` for the first time, use the arguments `--regenerate` and `--empty`.
```

Comment:
What's a difference between `--regenerate` and `--empty`?

Reply:
`--regenerate` makes it ignore any existing doughnut and look at what's on disk to create an up-to-date doughnut (useful if something messed up or if files have been processed without the doughnut). This can be really slow because it loops over the rows in the manifest and checks if files exist. So I added `--empty` which doesn't look at the disk and just makes an empty doughnut. I think I'll change it so that `--empty` doesn't require `--regenerate`

Reply:
Docs updated, making a new PR in https://github.com/neurodatascience/mr_proc to update the code